servers_v6.json

```
@@ -31,6 +31,10 @@
     "name": "Ranked",
     "address": ["mindustryranked.ddns.net:16567"]
   },
+  {
+    "name": "TWS",
+    "address": ["twsmindustry.24x7.hk:6701", "twsmindustry.24x7.hk:6702]
```

```suggestion
    "address": ["twsmindustry.24x7.hk:6701", "twsmindustry.24x7.hk:6702"]
```

Missing quote. This isn't valid JSON.